src/Extensions/LeftAndMainSubsites.php

```diff
@@ -88,7 +89,13 @@ public function sectionSites($includeMainSite = true, $mainSiteTitle = 'Main sit
         // Collect permissions - honour the LeftAndMain::required_permission_codes, current model requires
         // us to check if the user satisfies ALL permissions. Code partly copied from LeftAndMain::canView.
         $codes = [];
-        $extraCodes = Config::inst()->get(get_class($this->owner), 'required_permission_codes');
+        $ownerClass = get_class($this->owner);
```

Comment:
You can also use `$ownerClass` on row 96 (old)/103 (new).

Reply:
Upated